liesel/goose/optim.py

```diff
@@ -277,13 +297,17 @@ def optim_flat(
     batch_seed = (
         batch_seed if batch_seed is not None else np.random.randint(low=1, high=1000)
     )
-    model_test = model_test if model_test is not None else model
+
+    user_patience = stopper.patience
+    if model_validation is None:
+        model_validation = model_validation if model_validation is not None else model
```

Comment:
If `model_validation` is None, you can already assign `model_validation = model` without the inline if statement, right?

Reply:
Good point! 😅